app/models.py

```diff
@@ -0,0 +1,40 @@
+from uuid import UUID, uuid4
+from datetime import datetime, timezone
+from pydantic import BaseModel, EmailStr, Field
+from passlib.context import CryptContext
+
+#Set up password hashing context using bcrypt
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+
+class UserBase(BaseModel):
+    username: str
+    email: EmailStr
+    is_admin: bool = False
+    is_active: bool = True
+
+class UserCreate(UserBase):
+    password: str
+
+#Inheriting from UserBase
+class User(UserBase):
+    id: UUID = Field(default_factory=uuid4)
+    created_at: datetime = Field(default_factory=lambda: datetime.now(timezone.utc))
+    updated_at: datetime = Field(default_factory=lambda: datetime.now(timezone.utc))
```

Comment:
default value for the update date should be None (NULL)

Reply:
done.